pandas/io/formats/printing.py

```diff
@@ -497,11 +497,7 @@ def _justify(
     tail = [
         tuple(x.rjust(max_len) for x, max_len in zip(seq, max_length)) for seq in tail
     ]
-    # https://github.com/python/mypy/issues/4975
-    # error: Incompatible return value type (got "Tuple[List[Sequence[str]],
-    #  List[Sequence[str]]]", expected "Tuple[List[Tuple[str, ...]],
-    #  List[Tuple[str, ...]]]")
-    return head, tail  # type: ignore[return-value]
```

Comment:
Do I understand this correctly, this ignore comment was not necessary. Don't we have a check to report unused ignore comments?

Reply:
> Do I understand this correctly, this ignore comment was not necessary. 

Yes

> Don't we have a check to report unused ignore comments?

`pyproject.toml` includes `warn_unused_ignores = true`. I don't know why this wasn't picked up.

Reply:
You could probably rename the re-assigned varibles:
```py
head_tuple = [tuple(...) ...]
tail_tuple = [tuple(...) ...]
return head_tuple, tail_tuple
```
Mypy complains about the current version because it doesn't like to re-assign a new type (in this case `head` and `tail` are already typed in the function signature).